test/stubs/index.js

```diff
@@ -1,8 +1,52 @@
+const mongodb = require('mongodb');
+const EventModel = require('../../src/models/Event');
+const { MongoMemoryServer } = require('mongodb-memory-server-core');
+
+// TODO(Jason Katz-Brown) reset the DB between tests?
+let testDb = null;
+
+const connectToTestDatabase = () => {
```

Comment:
We should remove this just so we don't confuse anyone about what mongo connection we're relying on for tests